TeamCode/src/main/java/org/firstinspires/ftc/teamcode/controls/def/button/ButtonCtlTrigger.java

```diff
@@ -0,0 +1,8 @@
+package org.firstinspires.ftc.teamcode.controls.def.button;
+
+public enum ButtonCtlTrigger {
```

Comment:
Will this enum be used in more places than just ButtonCtlDefinition?

Reply:
no

Reply:
ikelt i klase maybe